utm.yaml

```diff
@@ -382,11 +386,11 @@ components:
           $ref: '#/components/schemas/SubscriptionID'
         version:
           description: |-
-            Version that the DSS changes every time a USS changes the subscription.  The DSS
-            incrementing the notification_index does not constitute a change that triggers a
-            new version.  A USS must specify this version when modifying an existing
-            subscription to ensure consistency in read-modify-write operations and distributed
-            systems.
+            Version of the Subscription that the DSS changes every time a USS changes the
```

Comment:
Any reason why this "Subscription" is capitalized?

Reply:
We used to capitalize the terms of art (Operational Intent, Constraint, Entity, Subscription, etc) and this one instance was missed when harmonizing with capitalizing only states -- good catch.  There do not appear to be any other instances of Subscription capitalization.  Note that "Entity" is still capitalized as a term of art; I'm inclined to leave that as-is.